.travis.yml

```diff
@@ -9,6 +9,9 @@ addons:
     packages:
       - libstdc++6
       - fonts-droid
+language: node_js
+node_js:
+  - "7"
```

Comment:
node.js homepage says that version 6.10.2 is LTS and recommended for most users. Current version is 7.9.0 which is the one we'll get if merging this change.